import-service/src/handlers/importFileParser.ts

```diff
@@ -0,0 +1,53 @@
+import { S3Event } from 'aws-lambda'
+import { buildResponse, getErrorBody } from '../utils'
+import {
+  CopyObjectCommand,
+  DeleteObjectCommand,
+  GetObjectCommand,
+  S3Client,
+} from '@aws-sdk/client-s3'
+import { parse } from 'csv-parse'
+
+export const importFileParser = async (event: S3Event) => {
+  console.log('importFileParser: ', event)
+
+  try {
+    const s3File = event.Records[0].s3
+    const client = new S3Client()
+    const getObjectCommandOutput = await client.send(
+      new GetObjectCommand({
+        Bucket: s3File.bucket.name,
+        Key: s3File.object.key,
+      })
+    )
+
+    await (getObjectCommandOutput?.Body as ReadableStream)
+      ?.pipe(parse())
```

Comment:
Hm.., 
- Is appropriate method in *Body* for transform to *ReadableStream*
 - But *ReadableStream* hasn't methot *pipe* (it's a method of *ReadStream* of Node), how it works..?